enclave_apps/oak_functions_enclave_app/src/main.rs

```diff
@@ -21,24 +21,13 @@
 extern crate alloc;
 
 use alloc::{boxed::Box, sync::Arc};
-use core::panic::PanicInfo;
 
-use log::info;
 use oak_core::samplestore::StaticSampleStore;
-use oak_restricted_kernel_sdk::{FileDescriptorChannel, StderrLogger};
-
-static LOGGER: StderrLogger = StderrLogger {};
-
-#[no_mangle]
-fn _start() -> ! {
-    log::set_logger(&LOGGER).unwrap();
-    log::set_max_level(log::LevelFilter::Debug);
-    oak_enclave_runtime_support::init();
-    main();
-}
+use oak_restricted_kernel_sdk::FileDescriptorChannel;
+use oak_restricted_kernel_sdk_proc_macro::entrypoint;
```

Comment:
can we export this macro from the SDK crate itself, instead of requiring two crates to be used?

Reply:
rust requires proc macros need to be in separate crates. We can re-export it from the SDK thought!

Reply:
yes that's what I meant, just export it from the SDK (I am aware it cannot be defined in here)

Reply:
done